src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -123,6 +126,39 @@ private void WriteClientMethodBody(CodeWriter writer, LowLevelClientMethod clien
             writer.Line();
         }
 
+        private string buildJsonFromSchemaDocs(string schemaName, LowLevelClientMethod.SchemaDocumentation[] docs)
+        {
+            var docDict = docs.ToDictionary(d => d.SchemaName, d => d);
+            IDictionary<string, Object> jsonObject = buildObjectFromSchemaDoc(docs.FirstOrDefault(), docDict, schemaName == LowLevelClientMethod.SchemaDocumentation.RequestBody);
+            var options = new JsonSerializerOptions()
```

Comment:
Let me try that and since we have more control of the format, do you think we can follow [Java](https://srnagarstorage.blob.core.windows.net/srnagartestcontainer/TextAnalyticsLLCJavadocs/apidocs/com/azure/ai/textanalytics/TextAnalyticsClient.html) to remove the double quotes for key and value, then it becomes:
```
{
  propertyA: valueA
  propertyB: "Choice1" | "Choice2 | "Choice 3"
}
```

Reply:
Yeah, following Java here and ending up with something like what you have above sounds good to me.

Reply:
Updated.